app/backend/src/couchers/servicers/communities.py

```diff
@@ -180,6 +180,59 @@ def ListAdmins(self, request, context):
                 next_page_token=str(admins[-1].id) if len(admins) > page_size else None,
             )
 
+    def AddAdmin(self, request, context):
+        with session_scope() as session:
+            node = session.execute(select(Node).where(Node.id == request.community_id)).scalar_one_or_none()
+            if not node:
+                context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+            if not can_moderate_node(session, context.user_id, node.id):
+                context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.NODE_MODERATE_PERMISSION_DENIED)
+
+            user = session.execute(
+                select(User).where_users_visible(context).where(User.id == request.user_id)
+            ).scalar_one_or_none()
+            if not user:
+                context.abort(grpc.StatusCode.NOT_FOUND, errors.USER_NOT_FOUND)
+
+            subscription = user.cluster_subscriptions.where(
+                ClusterSubscription.cluster_id == request.community_id
```

Comment:
Cluster id and node id are different. I think here you need `node.official_cluster.id`

Reply:
Oh I see, I wasn't paying close enough attention to the fact that `community_id` is a node id. I should've noticed that when just adding the `node` logic I found in the other functions.